Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -245,6 +245,15 @@
     std::optional<RequestedScrollData> requestedScroll;
 #endif
 
+    // Process any callbacks for unhiding content early, so that we
+    // set the root node during the same CA transaction.
+    for (auto& callbackID : layerTreeTransaction.callbackIDs()) {
+        if (callbackID == m_replyForUnhidingContent) {
+            m_replyForUnhidingContent = AsyncReplyID { };
```

Comment:
Weird that clearing something causes something else to be processed (I think the code before https://github.com/WebKit/WebKit/commit/359bfd90327f3b91f80398c53404a03e6a19eecd was a bit clearer), maybe we can give it a name that is clear that it's something we're waiting for? (so that "clearing it" is obviously "no longer waiting"?)